rust/agama-dbus-server/src/main.rs

```diff
@@ -11,7 +11,7 @@ const ADDRESS: &str = "unix:path=/run/agama/bus";
 async fn main() -> Result<(), Box<dyn std::error::Error>> {
     // When adding more services here, the order might be important.
     crate::questions::start_service(ADDRESS).await?;
-    crate::locale::start_service(ADDRESS).await?;
+    let _conn = crate::locale::start_service(ADDRESS).await?;
```

Comment:
btw we should document that we need to store connection, otherwise it will drop whole dbus object. So next time we do not remove it :)

Reply:
Yes, we should document it. Additionally, trying the approach I proposed in the description might help.